### DISCUSSION TITLE
Guess the number

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1070] [Galaxy Trivia] [answered]
- The first digit is a prime number.
- The second digit is the fifth digit minus the first digit.
- The third digit is twice the first digit.
- The fourth digit is the third digit plus three.
- The fifth digit is the difference between the first digit and the fourth digit.

**Comment by peterbaccaro** [comment, marked as answer]
The answer is **23,475**.
